portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,276 @@
+}
+#home p {
+  color: #fff;
+  font-size: 20px;
+}
+/* ------About page------ */
+#about {
+  overflow: hidden;
+  background: url("../images/aboutBackground.jpg");
+  background-size: cover;
+}
+#about::before {
+  content: "";
+  background: rgba(0, 0, 0, 0.3);
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  position: absolute;
+}
+.about-content {
+  position: relative;
+  font-size: 24px;
+  background-color: rgba(255, 255, 255, 0.15);
+  transform: translate(250px);
+  margin: 400px;
+  padding: 50px;
+  border-radius: 20px;
+  z-index: 2;
+}
+#about h1 {
+  color: #fff;
+  font-size: 60px;
+  position: absolute;
+  transform: translate(-350px, -300px);
+}
+#about p {
+  color: #fff;
 }
 
-#greeting-container {
+/* ------Education page------ */
+#education {
+  color: #fff;
+  background: url("../images/edu.jpg");
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+#education::before {
+  content: "";
+  background: rgba(0, 0, 0, 0.5);
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  position: absolute;
+}
+#education h1 {
+  font-size: 50px;
+  transform: translate(700px, -350px);
+  z-index: 100;
   margin-top: 20px;
 }
+.education-content {
+  margin-top: 80px;
+  margin-left: 180px;
+  z-index: 100;
+}
+.schs {
+  background-color: rgba(255, 255, 255, 0.15);
+  position: relative;
+  width: 1000px;
+  height: 200px;
+  padding: 10px;
+  padding-right: 50px;
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+}
+.left-container {
+  margin-left: 400px;
+  margin-top: 40px;
+}
+.left-container h2 {
+  transform: translateY(-20px);
+}
+.left-container p {
+  margin-top: 10px;
+  font-size: 18px;
+}
+.inti {
+  background-color: rgba(255, 255, 255, 0.15);
+  position: relative;
+  width: 1000px;
+  height: 200px;
+  padding: 10px;
+  margin-top: 20px;
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+}
+.right-container {
+  margin-left: 30px;
+  margin-right: 400px;
+  margin-top: 40px;
+}
+.right-container h2 {
+  transform: translateY(-20px);
+}
+.right-container p {
+  margin-top: 10px;
+  font-size: 18px;
+}
+.fresno {
+  background-color: rgba(255, 255, 255, 0.15);
+  position: relative;
+  width: 1000px;
+  height: 200px;
+  padding: 10px;
+  margin-top: 20px;
+  box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+}
+/* ------Experience page------ */
+#experience {
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: url("../images/exp.jpg");
+  background-size: cover;
+}
+#experience::before {
+  content: "";
+  background: rgba(0, 0, 0, 0.2);
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  position: absolute;
+}
+#experience h1 {
+  font-size: 50px;
+  margin-left: 200px;
+  z-index: 100;
+}
+
+/* Make it Responsive */
+/* @media all and (min-width: 1024px) {
+  h1 {
+    font-size: 28px;  
+  }
+  h2 {
+    font-size: 24px;
+  }
+  p {
+    font-size: 18px;
+    line-height: 24px;
+  }
+  q {
+    font-size: 24px;
+  }
+}  */
```

Comment:
Have you tried enabling those code to see if it would become responsive?

If it doesn't work, then maybe consider deleting it? What do you think?

Reply:
it did not work as expected so I commented it out to adjust it later but sure I can delete it.

Reply:
Hey Yii Choong, I think when you resolve this conversation, it will prompt me for another review.

Reply:
Hey Yii Choong, as mentioned in #5, it would be awesome to delete the comments in this PR instead. That way, when you merge this PR, it has all the changes that we've talked about in this branch!

Here are the general steps you can follow:
1. checkout branch `YcDevelopment`
2. delete the comments from the `YcDevelopment` branch
3. push changes to Github
4. resolve this comment
5. delete PR #5